app/Http/Controllers/EntityCreatorController.php

```diff
@@ -95,9 +97,18 @@ public function store(Request $request, Campaign $campaign, EntityType $entityTy
         // Handle dynamic elements
         $this->inputFields = $values;
         $this->dynamicTags()
+            ->dynamicParent($entityType)
+            ->dynamicLocations()
             ->dynamicLocation();
+        if ($entityType->code == 'character') {
```

Comment:
Better to do `$entityType->id == config('entities.ids.character')` in case we change the module name one day